web/src/features/filtering/components/ObserverStatusSelect.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { useFilteringContainer } from '../hooks/useFilteringContainer';
+import { GenericSelectFilter, GenericSelectFilterOption } from './GenericSelectFilter';
+
+const observerStatusOptions: GenericSelectFilterOption[] = [
```

Comment:
name should be `monitoringObserverStausOptions` since this filer will be used by ngo admins 
rename component file as well